controllers/providers/dns/infoblox.go

```diff
@@ -70,14 +70,14 @@ func (p *InfobloxProvider) CreateZoneDelegationForExternalDNS(gslb *k8gbv1beta1.
 		m.InfobloxIncrementZoneUpdateError(gslb)
 		return err
 	}
-	var delegateTo []ibclient.NameServer
+	var delegateTo []ibcl.NameServer
 
 	for _, address := range addresses {
-		nameServer := ibclient.NameServer{Address: address, Name: p.config.GetClusterNSName()}
+		nameServer := ibcl.NameServer{Address: address, Name: p.config.GetClusterNSName()}
 		delegateTo = append(delegateTo, nameServer)
 	}
 
-	findZone, err := objMgr.GetZoneDelegated(p.config.DNSZone)
+	findZone, err := p.getZoneDelegated(objMgr, p.config.DNSZone)
```

Comment:
Is this change related to ibclient version?

Reply:
nope, there is a new private function w/ almost the same signature that makes the actual call (`objMgr.GetZoneDelegated`) but this new function surrounds the call w/ time measurement so that we measure the request times in the histogram.